### PR TITLE
Feature/337: Fix Composer path.

### PR DESCRIPTION
**Pull request by tormi**
## Changes

- resolves https://github.com/wunderio/drupal-project/issues/337,
- sets Composer process-timeout config to 0, request by @misterjoonas.

### CODE REVIEW

**Comment by tormi** [comment]
Now we're using Composer version defined in https://github.com/drupal/core-dev/blob/10.0.x/composer.json and installed under `vendor` folder.
Screenshot 2023-02-08 at 14 40 37

File: .lando.yml [hunk 1]
```diff
@@ -6,12 +6,15 @@ config:
...
-  composer_version: 2
   xdebug: off
   config:
     php: .lando/php.ini
 
 tooling:
+  composer:
+    description: "Runs Composer commands"
+    cmd:
+      - appserver: ./vendor/bin/composer
```
Comment (misterjoonas) [inline .lando.yml:17]: This doesn't work if I need to remove vendor (every time composer hangs)
```
rm -rf vendor
❯ lando composer install
OCI runtime exec failed: exec failed: unable to start container process: exec: "./vendor/bin/composer": stat ./vendor/bin/composer: no such file or directory: unknown
```

Comment (tormi) [reply .lando.yml:17]: Looks like @tparssinen has proper fix in place here: https://github.com/wunderio/drupal-project/pull/329/files#diff-307b4548b91e997f23c35d90ed5e9a044e156b90425c9923d6b85d7dc11e67e2R22

Comment (tormi) [reply .lando.yml:17]: Committing update soon.

**Review by misterjoonas — APPROVED** [review]
This looks great to me. Tested the composer changes by rebuilding lando a couple of times and also `rm -rf vendor && lando composer install` works.

**Pull request merged**